srcpkgs/rng-tools/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'rng-tools'
 pkgname=rng-tools
 version=6.10
-revision=3
+revision=4
```

Comment:
Is a revbump necessary? It depends on `jitterentropy` right now:

```
~ ➜ xbps-query -Rx rng-tools
musl>=1.1.21_1
libsysfs>=2.1.0_1
jitterentropy>=3.0.0_1
jansson>=2.4_1
libcurl>=7.19_1
libcrypto46>=3.1.1_1
```

Having the patch is good, I just want to avoid an unnecessary rebuild.

Reply:
It is necessary! This issue doesn't happen on musl :) So the packages differ between the two libcs.